sklearn/multioutput.py

```diff
@@ -246,6 +245,12 @@ class MultiOutputRegressor(RegressorMixin, _MultiOutputEstimator):
     estimators_ : list of ``n_output`` estimators
         Estimators used for predictions.
 
+    Notes
+    -----
+    When individual estimators are fast to train or predict,
+    using ``n_jobs > 1`` can result in slower performance due
+    to the parallelism overhead.
+
```

Comment:
I think placing this in the `n_jobs` description would make this information more visible.